build.cake

```diff
@@ -125,6 +125,48 @@ Task("Test")
             }
         );
     });
+
+Task("TestAllGuidRepresentations")
```

Comment:
This doesn't run on Evergreen, but I was using it locally.

It takes a very long time to run because it runs the full test suite 6 times.